assets/js/modules/adsense/index.js

```diff
@@ -193,3 +201,44 @@ export const registerWidgets = ( widgets ) => {
 		[ AREA_MAIN_DASHBOARD_MONETIZATION_PRIMARY ]
 	);
 };
+
+export const ADSENSE_NOTIFICATIONS = {
+	'adsense-abr-success-notification': {
+		Component: AdBlockingRecoverySetupSuccessNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			const isConnected =
+				select( CORE_MODULES ).isModuleConnected( 'adsense' );
+			if ( ! isConnected ) {
+				return false;
+			}
+
+			await resolveSelect( MODULES_ADSENSE ).getSettings();
+			const adBlockingRecoverySetupStatus = await select(
+				MODULES_ADSENSE
+			).getAdBlockingRecoverySetupStatus();
```

Comment:
Also, it doesn't hurt but we shouldn't be using `await` on `select` (only `resolveSelect`).